tests/testthat/test-derive_locf_records.R

```diff
@@ -0,0 +1,158 @@
+test_that("derive_locf_records Test 1: new locf observations are derived correctly when visits are missing", {
```

Comment:
Please call the "Format test_that test file" addin to add the standard comments. See https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#addin-admiraldevformat_test_that_file.